pkg/controller/handlers/toolreference/toolreference.go

```diff
@@ -467,6 +470,7 @@ type mcpServerConfig struct {
 	Args           []string          `json:"args,omitempty"`
 	HTTPHeaders    []types.MCPHeader `json:"httpHeaders,omitempty"`
 	URL            string            `json:"url,omitempty"`
+	Remote         bool              `json:"remote,omitempty"`
```

Comment:
This field was added to support remote MCP servers with no default URL (which we wanted for Zapier).